rules/windows/privilege_escalation_uac_bypass_dll_sideloading.toml

```
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2020/10/27"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies User Account Control (UAC) bypass via DLL side loading. Attackers bypass UAC to stealthily execute code with
+elevated permissions.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Bypass UAC via Privileged IFileOperation COM interface"
+references = ["https://github.com/hfiref0x/UACME"]
+risk_score = 71
```

I think `high` severity rules should have a `risk_score` of 71.

```suggestion
risk_score = 73
```